File: marko_petriakov/bridge/src/model/implementShape/Circle.java
package model.implementShape;

import model.Color;
import model.Shape;

public class Circle implements Shape {

    private float radius;

    private Color color;

    public Circle(float radius, Color color){this.radius = radius; this.color=color;}
    @Override
    public float square() {
        return Math.round(Math.PI*Math.pow(radius,2));
    }

    @Override
    public float perimeter() {
        return Math.round(2*Math.PI*radius);
    }

    @Override
    public Color showColor() {
        return color;
    }

    public float getRadius() {
        return radius;
    }

    public void setRadius(float radius) {
        this.radius = radius;
    }
}

File: seman_anna/bridge/лаб3/Shape/Circle.cs
using System;

public class Circle : Shape
{
    protected double radius;

    public Circle(string color, double radius) : base(color)
    {
        this.radius = radius;
    }

    public override string Draw()
    {
        return $"Draw a {color} circle";
    }

    public override double CalculateArea()
    {
        return Math.PI * radius * radius;
    }

    public override double CalculatePerimeter()
    {
        return 2 * Math.PI * radius;
    }
}